scripts/deployment/hf_upload/cli.py

```
@@ -0,0 +1,104 @@
+    )
+    if not repo_id_uploaded:
+        return 1
+
+    # Step 5: Update deployment configs
+    try:
+        update_deployment_config(repo_id_uploaded, model_info, templates_dir)
+    except Exception as e:
+        logging.warning("Deployment config update failed: %s", e)
+
+    # Cleanup
+    if os.path.exists(temp_dir):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)